packages/commonwealth/client/scripts/state/api/communities/getCommuityById.ts

```diff
@@ -60,7 +60,7 @@ export const EXCEPTION_CASE_VANILLA_getCommunityById = async (
   // to @trpc/react-query v11.x
   const response = await axios.get(
     // eslint-disable-next-line max-len
-    `${SERVER_URL}/v1/community.getCommunity?batch=1&input=%7B%220%22%3A%7B%22id%22%3A%22${communityId}%22%2C%22include_node_info%22%3A${includeNodeInfo}%7D%7D`,
+    `${BASE_API_PATH}/community.getCommunity?batch=1&input=%7B%220%22%3A%7B%22id%22%3A%22${communityId}%22%2C%22include_node_info%22%3A${includeNodeInfo}%7D%7D`,
```

Comment:
should we prepend the server url here?
is this a temp fix to be replaced by the vanilla client soon?

Reply:
yes, i actually tried that but it seemed to create 2 trpc clients, anyways the ticket is there to handle this edgecase in future when the lib support is there.